### PR TITLE
create AL and MN yamls; update PA and WI yamls; delete WIL000565 yaml; delete empty files

### PR DESCRIPTION
**Pull request by OdeclasV**
Edit: throttle configs for `www.senate.mn` have been created

### CODE REVIEW

**Review by vramoscoreas — APPROVED** [review]

File: members/MNL000002.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          selector: "#city"
+          value: $ADDRESS_CITY
+          required: true
+        - name: address
+          selector: "#zip"
+          value: $ADDRESS_ZIP
+          required: true
+        - name: Subject
+          selector: "#subject"
+          value: $SUBJECt
```
Comment (tirish) [inline members/MNL000002.yaml:45]: Typo: 
```suggestion
          value: $SUBJECT
```
Pretty sure this matters

Comment (OdeclasV) [reply members/MNL000002.yaml]: oh wow thanks. Good catch

File: members/MNL000002.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          selector: "#street_address_1"
+          value: $ADDRESS_STREET
+          required: true
+        - name: street2
+          selector: "#street_address_2"
+          value: $ADDRESS_STREET_2
+          required: false
+        - name: street2
+          selector: "#street_address_2"
+          value: $ADDRESS_STREET_2
```
Comment (tirish) [inline members/MNL000002.yaml:29]: street 2 is in here twice

Comment (OdeclasV) [reply members/MNL000002.yaml]: fixed

File: members/ALL000001.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        - name: Recipient
+          selector: "#input_2_4"
+          value: "Gerald Allen"
+    - wait:
+        - value: 1
+    - click_on:
+        - value: Submit
+          selector: "#gform_submit_button_2"
+    - wait:
+        - value: 1
```
Comment (tirish) [inline members/ALL000001.yaml:32]: Should/could this be a `find` instead?

File: members/MNL000002.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+          required: false
+        - name: phone
+          selector: "#phone"
+          value: $PHONE_HYPHENS
+          required: false
+        - name: city
+          selector: "#city"
+          value: $ADDRESS_CITY
+          required: true
+        - name: address
```
Comment (tirish) [inline members/MNL000002.yaml:35]: minor: wrong `name` here, should be zip or postal code

**Review by tirish — APPROVED** [review]

File: members/ALL000001.yaml (hunk 3 above)
Comment (OdeclasV) [reply members/ALL000001.yaml]: updated

File: members/MNL000002.yaml (hunk 4 above)
Comment (OdeclasV) [reply members/MNL000002.yaml]: good catch. Updated

**Pull request merged**